python313Packages.imap-tools: 1.9.0 -> 1.9.1, paperless-ngx: 2.14.5 -> 2.14.6

dotlambda opened the pull request:
Diff: https://github.com/ikvk/imap_tools/compare/refs/tags/v1.9.0...v1.9.1

Changelog: https://github.com/ikvk/imap_tools/blob/v1.9.1/docs/release_notes.rst


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

dotlambda committed:
python313Packages.imap-tools: 1.9.0 -> 1.9.1

Diff: https://github.com/ikvk/imap_tools/compare/refs/tags/v1.9.0...v1.9.1

Changelog: https://github.com/ikvk/imap_tools/blob/v1.9.1/docs/release_notes.rst

NickCao commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 375682`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>paperless-ngx</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.deltachat</li>
    <li>python312Packages.deltachat.dist</li>
    <li>python312Packages.imap-tools</li>
    <li>python312Packages.imap-tools.dist</li>
    <li>python313Packages.deltachat</li>
    <li>python313Packages.deltachat.dist</li>
    <li>python313Packages.imap-tools</li>
    <li>python313Packages.imap-tools.dist</li>
  </ul>
</details>

dotlambda commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 375682`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>paperless-ngx</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.deltachat</li>
    <li>python312Packages.deltachat.dist</li>
    <li>python312Packages.imap-tools</li>
    <li>python312Packages.imap-tools.dist</li>
    <li>python313Packages.deltachat</li>
    <li>python313Packages.deltachat.dist</li>
    <li>python313Packages.imap-tools</li>
    <li>python313Packages.imap-tools.dist</li>
  </ul>
</details>

dotlambda commented:
We need to wait for https://github.com/paperless-ngx/paperless-ngx/commit/702de0cac3644e7a491ea47c5a6901e011324c6c to land in our version of paperless-ngx.

dotlambda committed:
paperless-ngx: 2.14.5 -> 2.14.6

Diff: https://github.com/paperless-ngx/paperless-ngx/compare/refs/tags/v2.14.5...v2.14.6

Changelog: https://github.com/paperless-ngx/paperless-ngx/releases/tag/v2.14.6

SuperSandro2000 committed:
paperless-ngx: disable flaky test

SuperSandro2000 commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 375682`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 9 packages built:</summary>
  <ul>
    <li>paperless-ngx</li>
    <li>python312Packages.deltachat</li>
    <li>python312Packages.deltachat.dist</li>
    <li>python312Packages.imap-tools</li>
    <li>python312Packages.imap-tools.dist</li>
    <li>python313Packages.deltachat</li>
    <li>python313Packages.deltachat.dist</li>
    <li>python313Packages.imap-tools</li>
    <li>python313Packages.imap-tools.dist</li>
  </ul>
</details>

SuperSandro2000 commented:
deployed to me server and paperless works as expected ✅
